examples/test-guestbook-spec-unused.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: helmrelease-spec-test
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: HelmRelease
+metadata:
+  name: guestbook
+  namespace: helmrelease-spec-test
+repo:
+  chartName: nginx-chart
+  source:
+    github:
+      branch: gh-pages
+      chartPath: nginx-chart
+      urls:
+        - https://github.com/pezhang/helm-release-API-test.git
```

Comment:
I don't think we should point at personal repo like this.

Could you not upload the charts you need to this repo first then reference this repo?

Reply:
@mikeshng Thanks for the review. Should I use this one: https://raw.github.com/open-cluster-management/multicloud-operators-subscription-release/master/test/helmrepo/nginx-ingress-1.40.0_keep.tgz ? Or do you have any other suggestion? Thanks a lot.

Reply:
I think you should make a separate PR with the chart that you want to use. We merge that PR first. Then you will have a file that you can reference in these PRs. Thanks.

Reply:
@mikeshng Ｉ got your point now. Will send the PR and change this code then. Thanks a lot.